python/GafferSceneUI/ShaderTweaksUI.py

```diff
@@ -81,6 +81,16 @@
 
 		],
 
+		"localise" : [
+
+			"description",
+			"""
+			Turn on to allow location-specific tweaks to be made to inherited
+			shaders. Shaders will be localised to locations matching the
+			node's filter prior to tweaking. The original inherited shader will
+			remain untouched.
+			"""
+		],
```

Comment:
Should we add `layout:index` metadata to avoid this coming awkwardly below the expanding + section?

![image](https://user-images.githubusercontent.com/1133871/78026465-96efd000-7353-11ea-8ed4-6dee7706f4af.png)

Or perhaps just add the `localise` plug before the `tweaks` plug in the constructor?